src/gui/optionsdialog.ui

```diff
@@ -972,10 +972,10 @@
                   <item>
                    <widget class="QCheckBox" name="checkConfirmMergeTrackers">
                     <property name="toolTip">
-                     <string>Shows a confirmation dialog upon merging trackers to existing torrent</string>
+                     <string>Shows a question dialog about merging trackers to existing torrent</string>
```

Comment:
There are 3 strings with "confirmation dialog" and 0 "question dialog".

Reply:
As per https://github.com/qbittorrent/qBittorrent/pull/19305#issuecomment-1634721561 I want to avoid term "confirm" in this case since it doesn't confirm some action but ask for action. IMO, it would be appropriate to "confirm the merge" if it only requested when the "merge" is enabled by default.
In fact, the current option caption does not require explanation, so I will delete this tooltip too.

Reply:
> As per [#19305 (comment)](https://github.com/qbittorrent/qBittorrent/pull/19305#issuecomment-1634721561) I want to avoid term "confirm" in this case since it doesn't confirm some action but ask for action (independently from another setting).

Of course I can turn it to be dependent from "Merge trackers to existing torrent" so it would really provide "confirm" behavior. But does it make sense to limit it in this way? Couldn't someone want to have it disabled for all non-interactive cases, but leave themselves the option when manually adding a torrent?

Reply:
Okay the change was very deliberate then, i was just pointing something out.
>Couldn't someone want to have it disabled for all non-interactive cases, but leave themselves the option when manually adding a torrent?

Yes of course.